### ISSUE TITLE
服务被终止

### ISSUE THREAD

**Issue by Wangzhipeng777** [yolain/ComfyUI-Easy-Use#584]
在使用easy imageRemBg 节点时Comfyui服务被直接终止，不清楚发生了什么，目前定位到是ComfyUI/custom_nodes/ComfyUI-Easy-Use/py/briaai/rembg.py  中的 BriaRMBG 中 推理代码         self.conv_in = nn.Conv2d(in_ch,64,3,stride=2,padding=1)
首行就被强行终止。
终止信息：
./run.sh: line 17: 314457 Aborted                 (core dumped) python main.py --listen 0.0.0.0 --cuda-device $param1 --port $param2

使用pytorch
torch                     2.4.1+cu121
cuda                      12.1
